versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+     serviceaccount/mongodb-enterprise-database-pods configured
+     Warning: resource serviceaccounts/mongodb-enterprise-ops-manager is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
+     serviceaccount/mongodb-enterprise-ops-manager configured
+     Warning: resource roles/mongodb-enterprise-appdb is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
+     role.rbac.authorization.k8s.io/mongodb-enterprise-appdb configured
+     Warning: resource rolebindings/mongodb-enterprise-appdb is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
+     rolebinding.rbac.authorization.k8s.io/mongodb-enterprise-appdb configured
+     WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     serviceaccount/mongodb-enterprise-appdb created
+     serviceaccount/mongodb-enterprise-database-pods created
+     serviceaccount/mongodb-enterprise-ops-manager created
+     role.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     rolebinding.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     serviceaccount/mongodb-enterprise-appdb created
+     serviceaccount/mongodb-enterprise-database-pods created
+     serviceaccount/mongodb-enterprise-ops-manager created
+     role.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     rolebinding.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     ```
+
+13. Switch to the master cluster and generate Ops Manager API keys and add our IP addresses to the Ops Manager access list using the following steps:
+
+   1. Get the Ops Manager URL using the following command:
```

Comment:
Some indentation issue here; they don't appear as sub-steps in HTML:

![image](https://github.com/user-attachments/assets/7b02413a-8cd2-469d-bb99-130d4cfa38d9)